var/spack/repos/builtin/packages/py-regex/package.py

```diff
@@ -25,4 +27,5 @@ class PyRegex(PythonPackage):
         "2017.07.11", sha256="dbda8bdc31a1c85445f1a1b29d04abda46e5c690f8f933a9cc3a85a358969616"
     )
 
-    depends_on("py-setuptools", type="build", when="@:2018,2020:")
+    depends_on("py-setuptools", when="@:2018,2020:", type="build")
```

Comment:
```suggestion
    depends_on("py-setuptools", type="build")
```
No idea why the dep was the way it was, but all versions use setuptools.